### ISSUE TITLE
The SayThanks link doesn't work

### ISSUE THREAD

**Issue by rocboronat** [JHBitencourt/ready_to_go#3]
So I'm leaving an issue to notify it and also to say thanks! 👯‍♂

**Comment by JHBitencourt** [comment]
I appreciate it.
Thanks for the support!

**Comment by JHBitencourt** [comment]
I appreciate it.
Thanks for the support!

**Comment by rocboronat** [comment]
Hey @JHBitencourt! But the button doesn't work, actually. You should remove it from the README file 😅

**Comment by JHBitencourt** [comment]
@rocboronat you're right. There is an open issue on their repo though https://github.com/BlitzKraft/saythanks.io/issues/60.

**Comment by rocboronat** [comment]
Oh! Great!
